docs/src/components/hardware-list.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import hardware from "../data/hardware.json";
+
+export default function HardwareList() {
+  return (
+    <>
+      <h3>Standalone Boards</h3>
+      <ul>
+        {hardware.standalone.map((p) => (
+          <li key={p.identifier}>
+            <a href={p.url}>{p.name}</a> (<code>{p.identifier}</code>)
+          </li>
+        ))}
+      </ul>
+      <span />
+      {hardware.compatible.map((h) => (
+        <React.Fragment key={h.name}>
+          <span />
+          <h3>{h.name}</h3>
+
+          <h4>Boards</h4>
```

Comment:
Thoughts on how to handle split wired not working yet?

Reply:
Could check if there's a components list and split them apart? Although The formatting might get weird. Maybe instead we just make a big disclaimer and get working on wired support? 😅